src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,307 @@
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to control the loading of images and iframes.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+Support for natively lazy-loading images and iframes is coming to the web! This video shows
+a [demo](https://mathiasbynens.be/demo/img-loading-lazy) of the feature:
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./lazyload.webm" type="video/webm">
+    <source src="./lazyload.mp4" type="video/mp4">
+  </video>
+</figure>
+
+Starting with Chrome 76, you'll be able to use the new `loading` attribute to lazy-load resources
+without the need to write custom lazy-loading code or use a separate JavaScript library. Let's dive
+into the details.
+
+## Why native lazy-loading?
+
+According to [HTTPArchive](https://httparchive.org/reports/page-weight), images are the most
+requested asset type for most websites and usually take up more bandwidth than any other
+resource. At the 90th percentile, sites send about 4.7 MB of images on desktop and mobile. That's a
+lot of [cat pictures](https://en.wikipedia.org/wiki/Cats_and_the_Internet).
+
+Embedded iframes also use a lot of data and can harm page performance. Only loading
+non-critical, below-the-fold images and iframes when the user is likely to see them improves
+page load times, minimizes user bandwidth, and reduces memory usage.
+
+Currently, there are two ways to defer the loading of off-screen images and iframes:
+
+- Using the [Intersection Observer
+  API](https://developers.google.com/web/updates/2016/04/intersectionobserver)
+- Using `scroll`, `resize`, or `orientationchange` [event
+  handlers](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/#using_event_handlers_the_most_compatible_way)
+
+Either option can let developers include lazy-loading functionality, and many developers have built
+third-party libraries to provide abstractions that are even easier to use. With lazy-loading
+supported directly by the browser, however, there's no need for an external library. Native lazy
+loading also ensures that deferred loading of images and iframes still works even if JavaScript is
+disabled on the client.
+
+## The `loading` attribute
+
+Today, Chrome already loads images at different priorities depending on where they're located with
+respect to the device viewport. Images below the viewport are loaded with a lower priority, but they're
+still fetched as soon as possible.
+
+In Chrome 76, you can use the `loading` attribute to completely defer the loading of offscreen images
+and iframes that can be reached by scrolling:
+
+```html
+<img src="image.png" loading="lazy" alt="…" width="200" height="200">
+<iframe src="https://example.com" loading="lazy"></iframe>
```

Comment:
`<iframe>` should also have `alt` attribute.

Reply:
What makes you say that? [`<iframe>` does not support `alt`.](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element)

Reply:
oh sorry, I meant `title`.

Reply:
per https://www.w3.org/TR/WCAG20-TECHS/H64.html

Reply:
ah good catch @Malvoz 

yeah it'd be nice to add this. If I recall, the screen reader will halt at the iframe and announce the title. The user can then choose to enter it using a keyboard shortcut or bypass the element.

Reply:
without the title my hunch is it may just read the src url of the iframe which may or may not be useful for the user.

Reply:
I found this https://web.dev/frame-title 😆